sphinx/ext/viewcode.py

```diff
@@ -75,10 +74,7 @@ def has_tag(modname, fullname, docname, refname):
                 env._viewcode_modules[modname] = False  # type: ignore
                 return
 
-            if not isinstance(analyzer.code, text_type):
-                code = analyzer.code.decode(analyzer.encoding)
-            else:
-                code = analyzer.code
+            code = analyzer.code
```

Comment:
How about moving this line after `find_tags()` call. I feel it is a bit better.

Reply:
Done :smile: